tests/common/startup/mod.rs

```diff
@@ -11,37 +10,34 @@ use common::data::{
 };
 
 use common::file_utils;
-use common::jcli_wrapper;
-use common::process_utils::process_guard::ProcessKillGuard;
-
+use common::jormungandr::{process::JormungandrProcess, starter};
 use std::path::PathBuf;
 
+use common::jcli_wrapper;
+
 pub use self::configuration_builder::ConfigurationBuilder;
 
-pub fn start_jormungandr_node(mut config: &mut JormungandrConfig) -> ProcessKillGuard {
-    jormungandr_starter::start_jormungandr_node(&mut config)
+pub fn start_jormungandr_node(mut config: &mut JormungandrConfig) -> JormungandrProcess {
+    starter::start_jormungandr_node(&mut config)
 }
 
-pub fn start_jormungandr_node_as_leader(mut config: &mut JormungandrConfig) -> ProcessKillGuard {
-    jormungandr_starter::start_jormungandr_node_as_leader(&mut config)
+pub fn start_jormungandr_node_as_leader(mut config: &mut JormungandrConfig) -> JormungandrProcess {
+    starter::start_jormungandr_node_as_leader(&mut config)
 }
 
-pub fn start_jormungandr_node_as_slave(mut config: &mut JormungandrConfig) -> ProcessKillGuard {
-    jormungandr_starter::start_jormungandr_node_as_slave(&mut config)
+pub fn start_jormungandr_node_as_slave(mut config: &mut JormungandrConfig) -> JormungandrProcess {
+    starter::start_jormungandr_node_as_slave(&mut config)
 }
 
 pub fn assert_start_jormungandr_node_as_passive_fail(
     mut config: &mut JormungandrConfig,
     expected_message_part: &str,
 ) {
-    jormungandr_starter::assert_start_jormungandr_node_as_passive_fail(
-        &mut config,
-        expected_message_part,
-    )
+    starter::assert_start_jormungandr_node_as_passive_fail(&mut config, expected_message_part)
 }
 
-pub fn start_jormungandr_node_as_passive(mut config: &mut JormungandrConfig) -> ProcessKillGuard {
-    jormungandr_starter::start_jormungandr_node_as_passive(&mut config)
+pub fn start_jormungandr_node_as_passive(mut config: &mut JormungandrConfig) -> JormungandrProcess {
+    starter::start_jormungandr_node_as_passive(&mut config)
```

Comment:
All these functions just delegate to the other module. Why not export those functions directly? You can use `pub use starter::start_jormungandr_node` to re-export.

Reply:
It's weird to get a mutable reference to a config structure and then pass it under another layer of mutable reference. What's that for?